ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -72,17 +72,15 @@ private[dao] final class TransactionsReader(
 
     val requestedRangeF = getEventSeqIdRange(startExclusive, endInclusive)
 
-    val query = (range: EventsRange[Long]) =>
-      (connection: Connection) => {
-        logger.debug(s"getFlatTransactions query($range)")
-        EventsTable
-          .preparePagedGetFlatTransactions(sqlFunctions)(
-            range = range,
-            filter = filter,
-            pageSize = pageSize,
-          )
-          .withFetchSize(Some(pageSize))
-          .asVectorOf(EventsTable.rawFlatEventParser)(connection)
+    val query = (range: EventsRange[Long]) => { implicit connection: Connection =>
```

Comment:
I personally dislike the idea of having implicit `java.sql.Connection`s passed around. It is not a typeclass and it is not a unique type.

Reply:
You're right, but that is how anorm's API is designed.

If you like, we can introduce a phantom type variable and subtype to hold it, so we can conform to anorm's API while having it be a "real" typeclass in our own code. But that ought to be a separate PR, I think.

Reply:
or just pass the implicit `Connection` explicitly everywhere in our code. But in any case it is a totally different topics, `anorm` encourages it and we do pass implicit `Connection` in a few other places.  Don't want to start another `implicit` vs explicit flame war :)

But I will definitely bring this up next time we have a team meeting.